jscomp/bsb/templates/react/src/simple/page.re

```diff
@@ -0,0 +1,13 @@
+let component = ReasonReact.statefulComponent "Greeting";
+
+let make ::name _children => {
+  let click _event state _self => ReasonReact.Update (state + 1);
+  {
+    ...component,
+    initialState: fun () => 0,
+    render: fun state self => {
+      let greeting = "Hello " ^ name ^ ". You've clicked the button " ^ (string_of_int state) ^ " time(s)!";
```

Comment:
could be improved as `{j|Hello $name , You've clicked the button $state times(s)! |j}`